k8s/Makefile

```diff
@@ -8,23 +8,57 @@ HELM_RUNTIME_BASE := helm-charts/seldon-core-v2-runtime/templates
+	sed -i 's/HACK_REMOVE_ME//' ${HELM_SERVERS_BASE}/._servers-statefulsets.tpl
+	sed -zi 's#\(apiVersion: rbac.authorization.k8s.io/v1\nkind: ClusterRole.*\- get\n\-\-\-\)#{{- if and (not .Values.controller.skipClusterRoleCreation) (or .Values.controller.clusterwide .Values.controller.watchNamespaces) -}}\n\1\n{{- end }}#' ${HELM_COMPONENTS_BASE}/_components-statefulsets.tpl
+	sed -zi 's#\(apiVersion: rbac.authorization.k8s.io/v1\nkind: ClusterRoleBinding.*ServiceAccount\n  name: seldon-v2-controller-manager\n  namespace:[^-]\{27\}\n\-\-\-\)#{{- if or .Values.controller.clusterwide .Values.controller.watchNamespaces -}}\n\1\n{{- end }}#' ${HELM_COMPONENTS_BASE}/_components-statefulsets.tpl
+	mv ${HELM_SERVERS_BASE}/._servers-statefulsets.tpl ${HELM_SERVERS_BASE}/_servers-statefulsets.tpl
+
+.PHONY: create-helm-charts-experimental
+create-helm-charts-experimental: kustomize
+	$(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone kustomize-experimental/helm-components-ns/ > ${HELM_COMPONENTS_BASE}/_components-deployments.tpl
+	$(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone kustomize-experimental/helm-servers/ > ${HELM_SERVERS_BASE}/_servers-deployments.tpl
+	sed -i 's/\(.*\)\(imagePullSecrets:\).*/{{- with .Values.imagePullSecrets }}\n\1\2\n\1{{- toYaml . | nindent 8 }}\n{{- end }}/g' ${HELM_COMPONENTS_BASE}/_components-deployments.tpl
+	sed -i 's/HACK_REMOVE_ME//' ${HELM_COMPONENTS_BASE}/_components-deployments.tpl
+	sed 's/labelz:/labels:\n\ \ \ /' ${HELM_SERVERS_BASE}/_servers-deployments.tpl > ${HELM_SERVERS_BASE}/._servers-deployments.tpl
+	sed -i 's/annotationz:/annotations:\n\ \ \ /' ${HELM_SERVERS_BASE}/._servers-deployments.tpl
+	sed -i 's/HACK_REMOVE_ME//' ${HELM_SERVERS_BASE}/._servers-deployments.tpl
+	sed -zi 's#\(apiVersion: rbac.authorization.k8s.io/v1\nkind: ClusterRole.*\- get\n\-\-\-\)#{{- if and (not .Values.controller.skipClusterRoleCreation) (or .Values.controller.clusterwide .Values.controller.watchNamespaces) -}}\n\1\n{{- end }}#' ${HELM_COMPONENTS_BASE}/_components-deployments.tpl
+	sed -zi 's#\(apiVersion: rbac.authorization.k8s.io/v1\nkind: ClusterRoleBinding.*ServiceAccount\n  name: seldon-v2-controller-manager\n  namespace:[^-]\{27\}\n\-\-\-\)#{{- if or .Values.controller.clusterwide .Values.controller.watchNamespaces -}}\n\1\n{{- end }}#' ${HELM_COMPONENTS_BASE}/_components-deployments.tpl
+	mv ${HELM_SERVERS_BASE}/._servers-deployments.tpl ${HELM_SERVERS_BASE}/_servers-deployments.tpl
+
+.PHONY: merge-configs
+merge-configs:
+	sed -i '1s;^;{{- define "setup.statefulsets" }}\n;' ${HELM_COMPONENTS_BASE}/_components-statefulsets.tpl && echo '{{- end }}' >> ${HELM_COMPONENTS_BASE}/_components-statefulsets.tpl
+	sed -i '1s;^;{{- define "setup.deployments" }}\n;' ${HELM_COMPONENTS_BASE}/_components-deployments.tpl && echo '{{- end }}' >> ${HELM_COMPONENTS_BASE}/_components-deployments.tpl
+	@{ \
+	echo '{{- if .Values.useServerWithDeployment }}'; \
```

Comment:
```suggestion
	echo '{{- if .Values.useDeploymentsForServers }}'; \
```

Reply:
Suggested the name change for increased clarity; It will need to be modified in a couple of other places as well.